src/packages/client/package.json

```diff
@@ -97,6 +110,7 @@
     "make-dir": "3.1.0",
     "mariadb": "2.5.2",
     "mssql": "6.3.1",
+    "ncp": "^2.0.0",
```

Comment:
We already have `@timsuchanek/copy` in this package, I guess we don't need `ncp` https://www.npmjs.com/package/ncp